### PR TITLE
Update IDE cell documentation, general IDE docs

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/1713

### CODE REVIEW

File: astro/cloud-ide/document-pipeline.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+sidebar_label: Document your Astro Cloud IDE pipeline
+title: Document your pipeline
+id: document-pipeline
+description: Learn how to add Markdown documentation to your pipeline and individual cells. 
+---
+
+Use Markdown cells to document your pipeline and make it easier for team members to collaborate on the same code. Your Markdown cell appears as a top-level comment in the generated DAG for your data pipeline. It does not appear in the Airflow UI and is not related to the the documentation features described in [Create DAG documentation in Apache Airflow](https://docs.astronomer.io/learn/custom-airflow-ui-docs-tutorial).
```
Comment (kwcanuck) [inline astro/cloud-ide/document-pipeline.md:8]:
```suggestion
Use Markdown cells to document your pipeline and make it easier for team members to collaborate on the same code. Your Markdown cell appears as a top-level comment in the generated DAG for your data pipeline. It does not appear in the Airflow UI and is not related to the features described in [Create DAG documentation in Apache Airflow](https://docs.astronomer.io/learn/custom-airflow-ui-docs-tutorial).
```

File: astro/cloud-ide/document-pipeline.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+## Create pipeline documentation
+
+1. In the pipeline editor, click **Add Cell**, then select **Markdown**.
+2. In the body of the cell, write your documentation in Markdown format. 
+
+To view your documentation in the Cloud IDE, click **Code**.
```
Comment (kwcanuck) [inline astro/cloud-ide/document-pipeline.md:15]:
```suggestion
3. Optional. Click **Code** to view your documentation in the Cloud IDE.
```

File: astro/cloud-ide/quickstart.md [hunk 3]
```diff
@@ -102,8 +102,8 @@ To create a SQL cell and execute SQL, first create a database to run your SQL qu
 
   :::info
 
-  SQL cell query results are stored as temporary tables in the database you're querying. The schema field in your connection object determines where these end up. Astronomer recommends setting this field to `TMP_ASTRO` for easy cleanup.
-
+  SQL cell query results are stored in XComs and are not accessible outside of your data pipeline. To save the results of a SQL query, run it in a SQL warehouse cell. See [Run SQL](cloud-ide/run-sql.md)
```
Comment (kwcanuck) [inline astro/cloud-ide/quickstart.md:105]:
```suggestion
 SQL cell query results are stored in XComs and are not accessible outside of your data pipeline. To save the results of a SQL query, run it in a SQL warehouse cell. See [Run SQL](cloud-ide/run-sql.md).
```

File: astro/cloud-ide/quickstart.md [hunk 4]
```diff
@@ -166,40 +165,12 @@ To create a potential dependency to a Python cell, the upstream Python cell must
...
-df['col_a'] = df['col_a'] + 1
-return df
-```
-
-### Use the results of a Python cell in a SQL cell
-
-You can make a SQL cell downstream dependent of a Python cell as long as the Python cell returns a pandas DataFrame. To do so, reference the name of the Python cell in your SQL cell using double curly braces, also known as jinja templating. Pandas DataFrames are automatically converted to SQL tables for use in SQL cells.
-
-The following SQL cell is downstream of a Python cell named `my_dataframe`.
+You can generate dependencies between any two types of cells. See the following documentation to learn more about each possible data dependency:
```
Comment (kwcanuck) [inline astro/cloud-ide/quickstart.md:168]:
```suggestion
You can generate dependencies between any two cell types. To learn more about data dependencies, see the following documentation:
```

File: astro/cloud-ide/document-pipeline.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+sidebar_label: Document your Astro Cloud IDE pipeline
+title: Document your pipeline
+id: document-pipeline
+description: Learn how to add Markdown documentation to your pipeline and individual cells. 
+---
+
+Use Markdown cells to document your pipeline and make it easier for team members to collaborate on the same code. Your Markdown cell appears as a top-level comment in the generated DAG for your data pipeline. It does not appear in the Airflow UI and is not related to the the documentation features described in [Create DAG documentation in Apache Airflow](https://docs.astronomer.io/learn/custom-airflow-ui-docs-tutorial).
+
+## Create pipeline documentation
+
```
Comment (kwcanuck) [inline astro/cloud-ide/document-pipeline.md:11]: Should you provide the path to the Cloud IDE? For example:

1. In the Cloud UI, select a Workspace and then select **Cloud IDE**.

File: astro/cloud-ide/quickstart.md [hunk 6]
```diff
@@ -63,12 +63,12 @@ After clicking **Create**, the IDE opens the pipeline editor. This is where you'
...
 
-Cells are the building blocks for pipelines. Cells can be created with either Python or SQL code. For this quickstart, you'll write a Python cell named `hello_world`.
+Cells are the building blocks for pipelines. They can complete a unit of work, such as a Python function or SQL query, or they can define assets for use throughout your pipeline. For this quickstart, you'll write a Python cell named `hello_world`.
 
 1. In the **Pipeline** list, click the name of the pipeline you created in step 2.
-2. Click **Add Cell** and select **Python**. A new cell named `cell_1` appears.
+2. Click **Add Cell** and select **Python**. A new cell named `python_1` appears.
 3. Click the cell's name and rename the cell `hello_world`.
-4. Click the editing window where it says `# Write your code here…`, then replace that line with the following code:
+4. Add the following line to the cell body:
```
Comment (kwcanuck) [inline astro/cloud-ide/quickstart.md:71]:
```suggestion
4. Add the following code to the cell:
```

File: astro/cloud-ide/document-pipeline.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
+---
+sidebar_label: Document your Astro Cloud IDE pipeline
+title: Document your pipeline
+id: document-pipeline
+description: Learn how to add Markdown documentation to your pipeline and individual cells.
```
Comment (kwcanuck) [inline astro/cloud-ide/document-pipeline.md:5]:
```suggestion
description: Learn how to add Markdown documentation to your pipeline and individual cells in the Astro Cloud IDE. 
```
Are users adding _documentation_, or _annotations_?

File: astro/cloud-ide/run-python.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
+---
+sidebar_label: Run Python in the Astro Cloud IDE
```
Comment (kwcanuck) [inline astro/cloud-ide/run-python.md:2]:
```suggestion
sidebar_label: Run Python
```

File: astro/cloud-ide/run-python.md [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+title: Run Python
```
Comment (kwcanuck) [inline astro/cloud-ide/run-python.md:3]:
```suggestion
title: Run Python in the Astro Cloud IDE
```

File: astro/cloud-ide/document-pipeline.md [hunk 10]
```diff
@@ -0,0 +1,15 @@
+---
+sidebar_label: Document your Astro Cloud IDE pipeline
```
Comment (kwcanuck) [inline astro/cloud-ide/document-pipeline.md:2]:
```suggestion
sidebar_label: Add Markdown documentation
```

File: astro/cloud-ide/document-pipeline.md [hunk 11]
```diff
@@ -0,0 +1,15 @@
...
+title: Document your pipeline
```
Comment (kwcanuck) [inline astro/cloud-ide/document-pipeline.md:3]:
```suggestion
title: Add Markdown documentation to an Astro Cloud IDE pipeline
```

File: astro/cloud-ide/run-python.md [hunk 12]
```diff
@@ -0,0 +1,73 @@
...
+id: run-python
+description: Learn how to run Python code by creating and configuring Python cells in the Astro Cloud IDE.
+---
+
+A Python cell contains a Python function that you can run in isolation or as a dependency in your pipeline. Create Python cells to execute Python as part of your data pipeline. 
+
+## Create a Python cell
+
```
Comment (kwcanuck) [inline astro/cloud-ide/run-python.md:11]: 1. In the Cloud UI, select a Workspace and then select **Cloud IDE**.

File: astro/cloud-ide/run-python.md [hunk 13]
```diff
@@ -0,0 +1,73 @@
...
+1. In the pipeline editor, click **Add Cell** and select **Python**. A new cell named `python_1` appears.
```
Comment (kwcanuck) [inline astro/cloud-ide/run-python.md:12]:
```suggestion
2. Select a project.

3. On the **Pipelines** page click a pipeline name to open the pipeline editor.

4, Click **Add Cell** > **Python**.
```

File: astro/cloud-ide/document-pipeline.md [hunk 14]
```diff
@@ -0,0 +1,15 @@
...
+id: document-pipeline
+description: Learn how to add Markdown documentation to your pipeline and individual cells. 
+---
+
+Use Markdown cells to document your pipeline and make it easier for team members to collaborate on the same code. Your Markdown cell appears as a top-level comment in the generated DAG for your data pipeline. It does not appear in the Airflow UI and is not related to the the documentation features described in [Create DAG documentation in Apache Airflow](https://docs.astronomer.io/learn/custom-airflow-ui-docs-tutorial).
+
+## Create pipeline documentation
+
+1. In the pipeline editor, click **Add Cell**, then select **Markdown**.
```
Comment (kwcanuck) [inline astro/cloud-ide/document-pipeline.md:12]:
```suggestion
2. Select a project.

3. On the **Pipelines** page click a pipeline name to open the pipeline editor.

4. Click **Add Cell** > **Markdown**.
```

File: astro/cloud-ide/run-python.md [hunk 15]
```diff
@@ -0,0 +1,73 @@
...
+2. Click the cell's name and rename the cell.
```
Comment (kwcanuck) [inline astro/cloud-ide/run-python.md:13]:
```suggestion
5. Click the cell name and enter a name for the cell.
```

File: astro/cloud-ide/run-python.md [hunk 16]
```diff
@@ -0,0 +1,73 @@
...
+3. Add your Python code to the body of the cell.
+
+## Run a Python cell
+
+Click **Run** to run a Python cell in the Cloud IDE and check that your Python code runs as expected. 
+
+After you run a cell, the **Logs** tab contains all logs generated by the cell run, including Airflow logs and Python errors. The **Results** tab appears if your cell runs successfully and contains the contents of your Python console.
+
+When you run a cell, the Cloud IDE sends a request to an isolated worker in the Astronomer-managed control plane. The worker executes your cell and returns the results to the Cloud IDE. Executing cells in the Cloud IDE is offered free of charge. For more information on execution, see [Execution](/astro/cloud-ide/reference/security.md#execution).
```
Comment (kwcanuck) [inline astro/cloud-ide/run-python.md:22]:
```suggestion
When you run a cell, the Astro Cloud IDE sends a request to an isolated worker in the Astronomer-managed control plane. The worker executes your cell and returns the results to the Cloud IDE. Executing cells in the Cloud IDE is offered free of charge. For more information on execution, see [Execution](/astro/cloud-ide/reference/security.md#execution).
```
Confirm you're using the naming convention consistently throughout the documentation set. I've seen _Cloud IDE_ and _Astro Cloud IDE_.

File: astro/cloud-ide/run-python.md [hunk 17]
```diff
@@ -0,0 +1,73 @@
...
+
+## Create explicit dependencies for a Python cell
+
+In a Python cell, click **Dependencies** and select a cell to make it an explicit upstream dependency of your Python cell. When you run your entire pipeline, the Python cell cannot begin running until the selected upstream cell finishes running.
```
Comment (kwcanuck) [inline astro/cloud-ide/run-python.md:26]:
```suggestion
In a Python cell, click **Dependencies** and select a cell to make it an explicit upstream dependency of your Python cell. When you run your entire pipeline, the Python cell cannot begin running until the selected upstream cell finishes running.
```
I was unable to locate **Dependencies**.

File: astro/cloud-ide/run-python.md [hunk 18]
```diff
@@ -0,0 +1,73 @@
...
+
+To make a Python cell an upstream dependency for another cell, click **Dependencies** for the other cell and select the name of your Python cell. 
+
+## Create data dependencies for a Python cell
+
+You can use the output of other cells in your project within a Python function. You define these dependencies in Python, and the Cloud IDE automatically renders the dependencies in your project code and in the **Pipeline** view of your project.
+
+### Pass a value from a Python cell to another Python cell
```
Comment (kwcanuck) [inline astro/cloud-ide/run-python.md:34]:
```suggestion
### Pass a value from one Python cell to another Python cell 
```

File: astro/cloud-ide/run-python.md [hunk 19]
```diff
@@ -0,0 +1,73 @@
...
+
+```python
+df = my_sql_cell # my_sql_cell is a SQL cell which gets converted to a pandas DataFrame by default
+df['col_a'] = df['col_a'] + 1
+return df
+```
+
+## View complete code for Python cells
+
+To view your Python cell within the context of an Airflow DAG, click **Code** to open your pipeline code. The Airflow DAG includes your Python function as well as all of the code required to run it on Airflow.
```
Comment (kwcanuck) [inline astro/cloud-ide/run-python.md:71]:
```suggestion
To view your Python cell within the context of an Airflow DAG, click **Code**. The Airflow DAG includes your Python function as well as all of the code required to run it on Airflow.
```

File: astro/cloud-ide/run-sql.md [hunk 20]
```diff
@@ -0,0 +1,73 @@
+---
+sidebar_label: Run SQL in the Astro Cloud IDE
```
Comment (kwcanuck) [inline astro/cloud-ide/run-sql.md:2]:
```suggestion
sidebar_label: Run SQL

File: astro/cloud-ide/run-sql.md [hunk 21]
```diff
@@ -0,0 +1,73 @@
...
+title: Run SQL
```
Comment (kwcanuck) [inline astro/cloud-ide/run-sql.md:3]:
```suggestion
title: Run SQL in the Astro Cloud IDE
```

File: astro/cloud-ide/run-sql.md [hunk 22]
```diff
@@ -0,0 +1,73 @@
...
+description: Learn how to run SQL queries by creating and configuring SQL cells in the Astro Cloud IDE.
+---
+
+A SQL cell contains a SQL query that you can run in isolation or as a dependency in your pipeline. Create SQL cells to execute SQL as part of your data pipeline. 
+
+## Prerequisites 
+
+- A database connection. See [Step 5: Create a database connection](cloud-ide/quickstart.md#step-5-create-a-database-connection.md).
+
+To use Warehouse SQL cells, you also need write permissions to the data warehouse in your database connection.
```
Comment (kwcanuck) [inline astro/cloud-ide/run-sql.md:14]:
```suggestion
- For Warehouse SQL cells, you need write permissions to the data warehouse in your database connection. 
```

File: astro/cloud-ide/run-sql.md [hunk 23]
```diff
@@ -0,0 +1,73 @@
...
+A SQL cell contains a SQL query that you can run in isolation or as a dependency in your pipeline. Create SQL cells to execute SQL as part of your data pipeline. 
+
+## Prerequisites 
+
+- A database connection. See [Step 5: Create a database connection](cloud-ide/quickstart.md#step-5-create-a-database-connection.md).
+
+To use Warehouse SQL cells, you also need write permissions to the data warehouse in your database connection. 
+
+## Create a SQL cell
+
```
Comment (kwcanuck) [inline astro/cloud-ide/run-sql.md:17]:
```suggestion
1. In the Cloud UI, select a Workspace and then select **Cloud IDE**.
```

File: astro/cloud-ide/run-sql.md [hunk 24]
```diff
@@ -0,0 +1,73 @@
...
+1. In the pipeline editor, click **Add Cell** and select one of the following:
```
Comment (kwcanuck) [inline astro/cloud-ide/run-sql.md:18]:
```suggestion
2. Select a project.

3. On the **Pipelines** page click a pipeline name to open the pipeline editor.

4. Click **Add Cell** and select one of the follwoing options:

-   **SQL**: Runs a SQL query against a provided database connection and persists the results of the query in an XCom for use by other cells.

- **Warehouse SQL**: Runs a SQL query against a provided database connection and stores the results in your data warehouse.

```

File: astro/cloud-ide/run-sql.md [hunk 25]
```diff
@@ -0,0 +1,73 @@
...
+
+    - **SQL**: Runs a SQL query against a provided database connection and persists the results of the query in an XCom for use by other cells.
+    - **Warehouse SQL**: Runs a SQL query against a provided database connection and stores the results in your data warehouse.
+
+    Regardless of which cell type you choose, you can use the results of your queries in downstream SQL and Python cells.
+
+2. Click the cell's name and rename the cell.
```
Comment (kwcanuck) [inline astro/cloud-ide/run-sql.md:25]:
```suggestion
5. Click the cell name and enter a name for the cell.
```

File: astro/cloud-ide/run-sql.md [hunk 26]
```diff
@@ -0,0 +1,73 @@
...
+3. Add your SQL query to the body of the cell.
```
Comment (kwcanuck) [inline astro/cloud-ide/run-sql.md:26]:
```suggestion
6. Add your SQL query to the cell body.
```

File: astro/cloud-ide/run-sql.md [hunk 27]
```diff
@@ -0,0 +1,73 @@
...
+4. In the **Select Connection** list, select the connection for the database you want to query against.
```
Comment (kwcanuck) [inline astro/cloud-ide/run-sql.md:27]:
```suggestion
7. Click **Environment** and then click the **Connections** tab. 

8. Select the connection for the database you want to query against.
```
Maybe the UI has changed?

File: astro/cloud-ide/run-sql.md [hunk 28]
```diff
@@ -0,0 +1,73 @@
...
+5. Warehouse SQL cells only. Configure the **Output Table** for storing the results of your query. This table must be accessible from your configured connection.
+
+## Run a SQL cell
+
+Click **Run** to run a SQL cell in the Cloud IDE and check that your SQL query runs as expected. 
+
+After you run a cell, the **Logs** tab contains all logs generated by the cell run, including Airflow logs and Python errors. The **Results** tab appears if your cell runs successfully and contains the contents of the SQL table created by your query.
+
+When you run a cell, the Cloud IDE sends a request to an isolated worker in the Astronomer-managed control plane. The worker executes your cell and returns the results to the Cloud IDE. Executing cells in the Cloud IDE is offered free of charge. For more information on execution, see [Execution](/astro/cloud-ide/reference/security.md#execution).
```
Comment (kwcanuck) [inline astro/cloud-ide/run-sql.md:36]:
```suggestion
When you run a cell, the Cloud IDE sends a request to an isolated worker in the Astronomer managed control plane. The worker executes your cell and returns the results to the Cloud IDE. Executing cells in the Cloud IDE is free. For more information on execution, see [Execution](/astro/cloud-ide/reference/security.md#execution).
```

File: astro/cloud-ide/run-sql.md [hunk 29]
```diff
@@ -0,0 +1,73 @@
...
+
+For example, a SQL cell containing the following query is dependent on a SQL cell named `my_table`.
+
+```sql
+select * from {{my_table}} -- my_table is another SQL cell
+```
+
+## View complete code for SQL cells
+
+To view your SQL cell within the context of an Airflow DAG, click **Code** to open your pipeline code. The Airflow DAG includes your SQL query as well as all of the code required to run it on Airflow.
```
Comment (kwcanuck) [inline astro/cloud-ide/run-sql.md:71]:
```suggestion
To view your SQL cell within the context of an Airflow DAG, click **Code**. The Airflow DAG includes your SQL query as well as all of the code required to run it on Airflow.
```

File: astro/cloud-ide/run-sql.md [hunk 30]
```diff
@@ -0,0 +1,73 @@
...
+
+SQL cells use the Astro SDK to execute your queries. A standard SQL cell executes your query using `aql.run_raw_sql` and stores the results in XComs. A warehouse SQL cell runs your SQL query using `aql.transform` and loads the results in your data warehouse. See [Astro SDK documentation](https://astro-sdk-python.readthedocs.io/en/stable/astro/sql/operators/raw_sql.html).
```
Comment (kwcanuck) [inline astro/cloud-ide/run-sql.md:73]:
```suggestion
SQL cells use the Astro SDK to execute your queries. A standard SQL cell executes your query using `aql.run_raw_sql` and stores the results in XComs. A Warehouse SQL cell runs your SQL query using `aql.transform` and loads the results in your data warehouse. See [Astro SDK documentation](https://astro-sdk-python.readthedocs.io/en/stable/astro/sql/operators/raw_sql.html).
```

**Review by kwcanuck — APPROVED** [review]

File: astro/cloud-ide/document-pipeline.md [hunk 31]
```diff
@@ -0,0 +1,15 @@
+---
+sidebar_label: Document your Astro Cloud IDE pipeline
+title: Document your pipeline
+id: document-pipeline
+description: Learn how to add Markdown documentation to your pipeline and individual cells. 
+---
+
+Use Markdown cells to document your pipeline and make it easier for team members to collaborate on the same code. Your Markdown cell appears as a top-level comment in the generated DAG for your data pipeline. It does not appear in the Airflow UI and is not related to the the documentation features described in [Create DAG documentation in Apache Airflow](https://docs.astronomer.io/learn/custom-airflow-ui-docs-tutorial).
+
+## Create pipeline documentation
```
Comment (jwitz) [inline astro/cloud-ide/document-pipeline.md:10]:
```suggestion
## Prerequisites 

- An IDE project and pipeline. See [Step 2: Create a pipeline](cloud-ide.md/quickstart#step-2-create-a-pipeline).

## Create pipeline documentation
```

File: astro/cloud-ide/run-python.md (hunk 17 above)
Comment (jwitz) [reply astro/cloud-ide/run-python.md:26]: It's one of those annoying buttons without a screen indicator (the +- one). I need to write a GitHub issue for that in the UI

File: astro/cloud-ide/run-sql.md [hunk 32]
```diff
@@ -0,0 +1,73 @@
...
+id: run-sql
+description: Learn how to run SQL queries by creating and configuring SQL cells in the Astro Cloud IDE.
+---
+
+A SQL cell contains a SQL query that you can run in isolation or as a dependency in your pipeline. Create SQL cells to execute SQL as part of your data pipeline. 
+
+## Prerequisites 
+
+- A database connection. See [Step 5: Create a database connection](cloud-ide/quickstart.md#step-5-create-a-database-connection.md).
+
```
Comment (jwitz) [inline astro/cloud-ide/run-sql.md:13]:
```suggestion
```

File: astro/cloud-ide/run-python.md [hunk 33]
```diff
@@ -0,0 +1,79 @@
...
+
+1. In the Cloud UI, select a Workspace and then select Cloud IDE.
+
+2. Select a project.
+
+3. On the **Pipelines** page, click a pipeline name to open the pipeline editor.
+
+4. Click **Add Cell** > **Python**.
+5. Click the cell name and enter a name for the cell.
+6. Add your Python code to the cell body.
```
Comment (jwitz) [inline astro/cloud-ide/run-python.md:20]:
```suggestion
4. Click **Add Cell** > **Python**.

5. Click the cell name and enter a name for the cell.

6. Add your Python code to the cell body.
```

File: astro/cloud-ide/run-sql.md [hunk 34]
```diff
@@ -0,0 +1,72 @@
...
+- A database connection. See [Step 5: Create a database connection](cloud-ide/quickstart.md#step-5-create-a-database-connection.md).
+- For Warehouse SQL cells, you need write permissions to the data warehouse in your database connection. 
+
+## Create a SQL cell
+
+1. In the pipeline editor, click **Add Cell** and select one of the following:
+
+    - **SQL**: Runs a SQL query against a provided database connection and persists the results of the query in an XCom for use by other cells.
+    - **Warehouse SQL**: Runs a SQL query against a provided database connection and stores the results in your data warehouse.
```
Comment (jwitz) [inline astro/cloud-ide/run-sql.md:20]:
```suggestion
```

File: astro/cloud-ide/run-sql.md (hunk 27 above)
Comment (jwitz) [reply astro/cloud-ide/run-sql.md:26]: That's for configuring a connection, but for individual cells you configure the connection in the cell itself

File: astro/cloud-ide/run-sql.md [hunk 35]
```diff
@@ -0,0 +1,72 @@
...
+
+    Regardless of which cell type you choose, you can use the results of your queries in downstream SQL and Python cells.
+
+2. Click the cell's name and rename the cell.
+3. Add your SQL query to the body of the cell.
+4. In the **Select Connection** list, select the connection for the database you want to query against.
```
Comment (jwitz) [inline astro/cloud-ide/run-sql.md:26]:
```suggestion
7. In the **Select Connection** list, select the connection for the database you want to query against.
```

File: astro/cloud-ide/run-python.md [hunk 36]
```diff
@@ -0,0 +1,79 @@
+---
+sidebar_label: Run Python
+title: Run Python in the Astro Cloud IDE
+id: run-python
+description: Learn how to run Python code by creating and configuring Python cells in the Astro Cloud IDE.
+---
+
+A Python cell contains a Python function that you can run in isolation or as a dependency in your pipeline. Create Python cells to execute Python as part of your data pipeline.
```
Comment (jwitz) [inline astro/cloud-ide/run-python.md:8]:
```suggestion
A Python cell contains a Python function that you can run in isolation or as a dependency in your pipeline. Create Python cells to execute Python as part of your data pipeline. 

## Prerequisites 

- An IDE project and pipeline. See [Step 2: Create a pipeline](cloud-ide.md/quickstart#step-2-create-a-pipeline).
```

File: astro/cloud-ide/run-sql.md [hunk 37]
```diff
@@ -0,0 +1,79 @@
...
+
+3. On the **Pipelines** page click a pipeline name to open the pipeline editor.
+
+4. Click **Add Cell** and select one of the follwoing options:
+
+    -   **SQL**: Runs a SQL query against a provided database connection and persists the results of the query in an XCom for use by other cells.
+
+    - **Warehouse SQL**: Runs a SQL query against a provided database connection and stores the results in your data warehouse.
+
+
```
Comment (jwitz) [inline astro/cloud-ide/run-sql.md:27]:
```suggestion
```

File: astro/cloud-ide/run-sql.md [hunk 38]
```diff
@@ -0,0 +1,79 @@
...
+title: Run SQL in the Astro Cloud IDE
+id: run-sql
+description: Learn how to run SQL queries by creating and configuring SQL cells in the Astro Cloud IDE.
+---
+
+A SQL cell contains a SQL query that you can run in isolation or as a dependency in your pipeline. Create SQL cells to execute SQL as part of your data pipeline. 
+
+## Prerequisites 
+
+- A database connection. See [Step 5: Create a database connection](cloud-ide/quickstart.md#step-5-create-a-database-connection.md).
```
Comment (jwitz) [inline astro/cloud-ide/run-sql.md:12]:
```suggestion
- An IDE project and pipeline. See [Step 2: Create a pipeline](cloud-ide.md/quickstart#step-2-create-a-pipeline).
- A database connection. See [Step 5: Create a database connection](cloud-ide/quickstart.md#step-5-create-a-database-connection.md).
```

File: astro/cloud-ide/document-pipeline.md [hunk 39]
```diff
@@ -0,0 +1,24 @@
...
+1. In the Cloud UI, select a Workspace and then select Cloud IDE.
+
+2. Select a project.
+
+3. On the **Pipelines** page click a pipeline name to open the pipeline editor.
+
+4. Click **Add Cell** > **Markdown**.
+2. In the body of the cell, write your documentation in Markdown format. 
+
+5. Optional. Click **Code** to view your documentation within the context of an Airflow DAG.
```
Comment (jwitz) [inline astro/cloud-ide/document-pipeline.md:24]:
```suggestion

5. In the body of the cell, write your documentation in Markdown format. 

6. Optional. Click **Code** to view your documentation within the context of an Airflow DAG.
```

File: astro/cloud-ide/document-pipeline.md [hunk 40]
```diff
@@ -0,0 +1,25 @@
+---
+sidebar_label: Document your pipeline
+title: Add documentation to an Astro Cloud IDE pipeline
+id: document-pipeline
+description: Learn how to add documentation to your pipeline and individual cells in the Astro Cloud IDE. 
+---
+
+Use Markdown cells to document your pipeline and make it easier for team members to collaborate on the same code. Your Markdown cell appears as a top-level comment in the generated DAG for your data pipeline. It does not appear in the Airflow UI and is not related to the features described in [Create DAG documentation in Apache Airflow](https://docs.astronomer.io/learn/custom-airflow-ui-docs-tutorial).
+## Prerequisites
```
Comment (jwitz) [inline astro/cloud-ide/document-pipeline.md:9]:
```suggestion

## Prerequisites 
```

File: astro/cloud-ide/run-sql.md [hunk 41]
```diff
@@ -0,0 +1,78 @@
...
+
+A SQL cell contains a SQL query that you can run in isolation or as a dependency in your pipeline. Create SQL cells to execute SQL as part of your data pipeline. 
+
+## Prerequisites 
+
+- An IDE project and pipeline. See [Step 2: Create a pipeline](cloud-ide.md/quickstart#step-2-create-a-pipeline).
+- A database connection. See [Step 5: Create a database connection](cloud-ide/quickstart.md#step-5-create-a-database-connection.md).
+- For Warehouse SQL cells, you need write permissions to the data warehouse in your database connection. 
+
+## Create a SQL cell
```
Comment (jwitz) [inline astro/cloud-ide/run-sql.md:16]:
```suggestion
## Create a SQL cell

```

File: astro/cloud-ide/run-sql.md [hunk 42]
```diff
@@ -0,0 +1,78 @@
...
+    -   **SQL**: Runs a SQL query against a provided database connection and persists the results of the query in an XCom for use by other cells.
+
+    - **Warehouse SQL**: Runs a SQL query against a provided database connection and stores the results in your data warehouse.
+
+    Regardless of which cell type you choose, you can use the results of your queries in downstream SQL and Python cells.
+
+5. Click the cell name and enter a name for the cell.
+6. Add your SQL query to the cell body.
+7. In the **Select Connection** list, select the connection for the database you want to query against.
+8. If you're using Warehouse SQL cells, configure the **Output Table** for storing the results of your query. This table must be accessible from your configured connection.
```
Comment (jwitz) [inline astro/cloud-ide/run-sql.md:33]:
```suggestion
1. In the Cloud UI, select a Workspace and then select **Cloud IDE**.
2. Select a project.

3. On the **Pipelines** page click a pipeline name to open the pipeline editor.

4. Click **Add Cell** and select one of the follwoing options:

    -   **SQL**: Runs a SQL query against a provided database connection and persists the results of the query in an XCom for use by other cells.

    - **Warehouse SQL**: Runs a SQL query against a provided database connection and stores the results in your data warehouse.

    Regardless of which cell type you choose, you can use the results of your queries in downstream SQL and Python cells.

5. Click the cell name and enter a name for the cell.

6. Add your SQL query to the cell body.

7. In the **Select Connection** list, select the connection for the database you want to query against.

8. If you're using Warehouse SQL cells, configure the **Output Table** for storing the results of your query. This table must be accessible from your configured connection.
```

**Review by jlaneve — APPROVED** [review]
nice, thank you!

**Pull request merged**